cypress/page/products-list.page.ts

```diff
@@ -0,0 +1,19 @@
+class ProductListPage {
+  private buttonAdd: string;
+  private proceedToCheckout: string;  
+
+  constructor() {
+      this.buttonAdd = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
+      this.proceedToCheckout = "[style*=\"display: block;\"] .button-container > a"
+  }
+
+  public addingProduct(): void {
```

Comment:
addProduct or selectProduct would be a better name :) The name of methods should contain verbs in infinitive mode.